lib/preprocessor.py

```diff
@@ -392,7 +392,10 @@ def lua_eval(self, name, expr, silent=True):
         result = self.servers[name].admin(
             '%s%s' % (expr, self.delimiter), silent=silent
         )
-        result = yaml.safe_load(result)
+        try:
+            result = yaml.safe_load(result)
+        except AttributeError:
+            result = []
```

Comment:
TarantoolAsyncConnection returns `None` at `socket.error`. There are several comments here:

1. How can I reproduce the situation?
2. Please, don't use try-except, where `'foo' in my_dict` or `x is None` may be used. The exception handling is not cheap thing.
3. As in PR #228 and PR #230, we should not hide the error and report an empty result to the caller.
4. There is nothing called `kill_all_servers()`, please, change or clarify (it is about the commit message).